src/main/webapp/js/user-page-loader.js

```diff
@@ -95,4 +96,21 @@ function buildUI() {
   setPageTitle();
   showMessageFormIfViewingSelf();
   fetchMessages();
-}
\ No newline at end of file
+  fetchAboutMe();
+}
+
+/** Fetches about me data */
+function fetchAboutMe(){
+    const url = '/about?user=' + parameterUsername;
+    fetch(url).then((response) => {
+        return response.text();
+    }).then((aboutMe) => {
+        const aboutMeContainer = document.getElementById('about-me-container');
+        if(aboutMe == ''){
+            aboutMe = 'This user has not entered any information yet.';
+        }
+
+        aboutMeContainer.innerHTML = aboutMe;
```

Comment:
If the user can submit arbitrary `aboutMe` content, including content that might include `<script>` tags and you assign that content to `innerHTML`, it would allow one user to inject `<script>` tags that get displayed and executed for another user, thereby allowing one user to run code that is in the security/execution context of another user.

In other words, assigning to `innerHTML` here is a potential vector for a cross-site scripting (XSS) attack:
https://en.wikipedia.org/wiki/Cross-site_scripting

This is something that even many experienced engineers do by mistake.

To mitigate this, I would recommend one of the following options:

  - Don't support rich content (assign to `textContent` which will turn HTML tags into
    the textual equivalents rather than actually treating those tags like active HTML).

  - Convert untrusted user input into trusted HTML before assigning to `innerHTML`. For example,
    you could use a markdown library that guarantees that only certain kinds of HTML will be passed
    through, with certain kinds of HTML (such as scripts) automatically escaped/removed